bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
No - the debug and hexdump libs were the ones I replaced already (my replacement does nothing since debug is disabled at compile time). 

This one is the Unix socket library which uses JNI (and hence the lib you reference). I'm not sure how you want to handle this library - I know your views on JNI  for DT but if this library is ok with Eclipse then I think it is best to leave it in the bundle since it will be easier for people to use if they don't need to change the dbus config.

The lib I think comes from the same author as dbus Java. 

The other option is people need to include this jar manually - not a big deal I guess. 

Sent from my iPhone

> On 15 Dec 2015, at 11:00, Kai Kreuzer <notifications@github.com> wrote:
> 
> In bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties:
> 
> > @@ -0,0 +1,9 @@
> > +bin.includes = META-INF/,\
> > +               OSGI-INF/,\
> > +               .,\
> > +               about.html,\
> > +               lib/libdbus-java-2.7.jar,\
> > +               lib/unix-0.5.jar
> This is the library that you wanted to replace by your own implementation, right?
> 
> What is the libunix-java.so? Is that needed?
> 
> —
> Reply to this email directly or view it on GitHub.
>